_posts/0000-01-02-JayeshPatil47.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Don't take A right Decisions , take Decisions and make them right" - Mr Ratanji Tata

Reply:
Add quote